README.md

```diff
@@ -459,7 +459,7 @@ The objective of the project is to achieve a minimal score of 80 %.
 
 To generate the coverage report, run the following command:
 ```bash
-pytest --cov=living_documentation_generator --cov-fail-under=80 --cov-report=html
+pytest --cov=. --cov-fail-under=80 --cov-report=html
```

Comment:
1. Why is this command different to one in CI?
2. This command fails.

Reply:
Because in CI workflow, we just aiming to see, if we reached the 80 % low limit of the CI workflow.